cpp/zeder_to_zotero_importer.cc

```diff
@@ -332,17 +348,20 @@ unsigned ImportZederEntries(const Zeder::EntryCollection &zeder_entries, Harvest
                 if (skip_entry) {
                     if (new_section)
                         harvester_config->removeConfigSection(title);
-                    continue;
+                    goto next_entry;
```

Comment:
Wouldn't a break also work here?

Reply:
almost, but if the entry is skipped we want ++num_entries_imported to be skipped as well